cl/phase1/network/gossip_manager.go

```diff
@@ -264,6 +195,17 @@ func (g *GossipManager) onRecv(ctx context.Context, data *sentinel.GossipData, l
 		default:
 		}
 	}
+	if errors.Is(err, services.ErrIgnore) {
+		return nil
+	}
+	if err != nil {
+		l["at"] = "block service"
```

Comment:
i feel that put error handling on block srv here is somewhat tricky, becasue it's handled after jumping 2 switch cases. 
any reason why not move it to line 155?